src/core/ext/transport/chttp2/transport/chttp2_transport.c

```diff
@@ -2912,8 +3006,7 @@ grpc_chttp2_incoming_byte_stream *grpc_chttp2_incoming_byte_stream_create(
     grpc_exec_ctx *exec_ctx, grpc_chttp2_transport *t, grpc_chttp2_stream *s,
     uint32_t frame_size, uint32_t flags) {
   grpc_chttp2_incoming_byte_stream *incoming_byte_stream =
-      (grpc_chttp2_incoming_byte_stream *)gpr_malloc(
-          sizeof(*incoming_byte_stream));
+      gpr_malloc(sizeof(*incoming_byte_stream));
```

Comment:
not sure why this is to be removed. Was this a merge conflict?

Reply:
Probably missed in a merge... I'll fix